data/crc.yml

```
@@ -34,6 +34,15 @@ crc:
     "Find an app or service":
       selectors:
         - '{} button:contains("Find an app or service")'
+    "Apps & Services Dropdown - Ansible":
+      selectors:
+        - '{} [data-ouia-component-id="AllServices-Dropdown-Ansible"]'
+    "Apps & Services Dropdown - RHEL":
+      selectors:
+        - '{} [data-ouia-component-id="AllServices-Dropdown-RHEL"]'
+    "Apps & Services Dropdown - Openshift":
+      selectors:
+        - '{} [data-ouia-component-id="AllServices-Dropdown-Openshift"]'
```

```suggestion
    "Apps & Services Dropdown - Ansible":
      selectors:
        - '[data-ouia-component-id="AllServices-Dropdown-Ansible"]'
    "Apps & Services Dropdown - RHEL":
      selectors:
        - '[data-ouia-component-id="AllServices-Dropdown-RHEL"]'
    "Apps & Services Dropdown - Openshift":
      selectors:
        - '[data-ouia-component-id="AllServices-Dropdown-Openshift"]'
```

You can remove the scope for these since they're going to be present on every page. You should be able to delete your changes further down since this will capture all the clicks sitewide